tests/src/test/java/alluxio/testutils/master/MasterTestUtils.java

```diff
@@ -64,6 +69,19 @@ public static MasterRegistry createStandbyFileSystemMasterFromJournal() throws E
     return createFileSystemMasterFromJournal(false, null);
   }
 
+  /**
+   * Creates a new leader {@link FileSystemMaster} from journal along with its dependencies, and
+   * returns the master registry containing that master.
+   *
+   * @return a master registry containing the created {@link FileSystemMaster} master
+   */
+  public static MasterRegistry createSnapshotLeaderFileSystemMasterFromJournal() throws Exception {
+    String masterJournal = ServerConfiguration.get(PropertyKey.MASTER_JOURNAL_FOLDER);
+    String tempDir = Files.createTempDir().getAbsolutePath();
```

Comment:
Are we going to attempt to clean this temp dir after the test?

Reply:
same question

Reply:
The OS will automatically clean up the temporary folder.

Reply:
Typically temp folders in the OS stick around for a while. I would prefer trying to delete the temp dir ourselves.